src/client.h

```diff
@@ -40,28 +40,25 @@
 #include "util.h"
 #include "buffer.h"
 #include "signalhandler.h"
-#ifdef LLCON_VST_PLUGIN
-#    include "vstsound.h"
+
+#if defined( _WIN32 ) && !defined( JACK_REPLACES_ASIO )
+#    include "../windows/sound.h"
 #else
-#    if defined( _WIN32 ) && !defined( JACK_REPLACES_ASIO )
-#        include "../windows/sound.h"
+#    if ( defined( Q_OS_MACX ) ) && !defined( JACK_REPLACES_COREAUDIO )
+#        include "../mac/sound.h"
 #    else
-#        if ( defined( Q_OS_MACX ) ) && !defined( JACK_REPLACES_COREAUDIO )
-#            include "../mac/sound.h"
+#        if defined( Q_OS_IOS )
+#            include "../ios/sound.h"
 #        else
-#            if defined( Q_OS_IOS )
-#                include "../ios/sound.h"
+#            ifdef ANDROID
+#                include "../android/sound.h"
 #            else
-#                ifdef ANDROID
-#                    include "../android/sound.h"
-#                else
-#                    include "../linux/sound.h"
-#                    ifndef JACK_REPLACES_ASIO // these headers are not available in Windows OS
-#                        include <sched.h>
-#                        include <netdb.h>
-#                    endif
-#                    include <socket.h>
+#                include "../linux/sound.h"
+#                ifndef JACK_REPLACES_ASIO // these headers are not available in Windows OS
+#                    include <sched.h>
+#                    include <netdb.h>
 #                endif
+#                include <socket.h>
 #            endif
 #        endif
```

Comment:
This whole thing looks ugly. Isn't there anything like an else if construct?

Reply:
Style guide sayeth neigh.